src/screens/ENSConfirmRegisterSheet.js

```diff
@@ -101,6 +101,8 @@ export default function ENSConfirmRegisterSheet() {
 
   useEffect(() => startPollingGasFees(), [startPollingGasFees]);
 
+  useEffect(() => blurFields());
```

Comment:
What does this do? It doesn't have a dependency array, so it would be invoked on every render.